common/persistence/persistence-tests/executionManagerTest.go

```diff
@@ -700,7 +720,8 @@ func (s *ExecutionManagerSuite) TestUpdateWorkflow() {
 	s.Equal(int32(20), info1.WorkflowTimeout)
 	s.Equal(int32(13), info1.DecisionTimeoutValue)
 	s.Equal([]byte(nil), info1.ExecutionContext)
-	s.Equal(p.WorkflowStateCreated, info1.State)
+	s.Equal(p.WorkflowStateRunning, info1.State)
+	s.Equal(p.WorkflowCloseStatusNone, info0.CloseStatus)
```

Comment:
should be info1. similar for the following copy 

(please leave other error you found in this unchanged, I have a pending PR address those)